### PR TITLE
#157110851 updating muscle list

### PR DESCRIPTION
**Pull request by frankip**
#### What does this PR do?
It updates the muscles as well as their exercises and also their respective images
#### Description of Task to be completed?
The muscles list should be permanent in that when you load the project it comes with all the muscles predefines along with their exercises
#### How should this be manually tested?
after cloning and running up the project run 
`python manage.py loaddata ./wger/exercises/fixtures/muscles.json`
and 
`python manage.py loaddata ./wger/exercises/fixtures/exercises.json`

to load all the muscles and exercise to the template 
#### Any background context you want to provide?
There is one muscle which I didn't need to add because it was already there `Obliquus externus abdominis`

#### What are the relevant pivotal tracker stories?
[#157110851](https://www.pivotaltracker.com/story/show/157110851)

#### Screenshots
screen shot 2018-05-03 at 18 08 12
screen shot 2018-05-03 at 18 10 23

37091554-ac8c697e-2219-11e8-88d4-fb54098cd7c5

### CODE REVIEW

File: wger/exercises/fixtures/muscles.json [hunk 1]
```diff
@@ -1,130 +1,155 @@
...
+  },
+  {
+    "pk": 19,
+    "model": "exercises.muscle",
+    "fields": {
+      "is_front": true,
+      "name": "Forearms"
+    }
+  }
+
```
Comment (Kalela) [inline wger/exercises/fixtures/muscles.json:154]: Great job implementing your task. What do you think about adding tests for the functionality?

File: wger/exercises/fixtures/muscles.json [hunk 2]
```diff
@@ -1,130 +1,155 @@
...
+  },
+  {
+    "pk": 19,
+    "model": "exercises.muscle",
+    "fields": {
+      "is_front": false,
+      "name": "Forearms"
+    }
+  }
+
```
Comment (mutoromike) [inline wger/exercises/fixtures/muscles.json]: Great job here. I think you could have made it even better by adding comments.

**Comment by mutoromike** [comment]
Great work adding the muscle lists. We could also do with comments.

File: wger/exercises/fixtures/muscles.json (hunk 1 above)
Comment (frankip) [reply wger/exercises/fixtures/muscles.json:154]: they have already been tested I don't need to test them, I was just extending them

File: wger/core/static/images/muscles/main/muscle-16.svg [hunk 3]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (meshnesh) [inline wger/core/static/images/muscles/main/muscle-16.svg:1]: Nice Implementation of adding image svg

**Review by meshnesh — APPROVED** [review]
Nice Implementation of the adding of muscles and adding exercises.

**Review by mutoromike — COMMENTED** [review]
Great work adding the muscle lists. We could also do with comments.

**Review by mutoromike — APPROVED** [review]

**Comment by betsybeth** [comment]
Good  work on the styling

**Pull request merged**